README.md

```diff
@@ -272,7 +273,7 @@ Brukeren gjennomfører signeringsseremonien, og blir deretter sendt tilbake til
 
 #### Steg 3: hent status
 
-Når brukeren blir sendt tilbake til din portal skal du gjøre et bak-kanal-kall (`HTTP GET`) for å hente ned status. Dette gjøres ved å benytte `status-url` du fikk i steg 1, pluss query-parameter (`status_query_token`) du fikk i steg 2.
+Når brukeren blir sendt tilbake til din portal skal du gjøre et bak-kanal-kall (`HTTP GET`) for å hente ned status. Dette gjøres ved å benytte `status-url` du fikk i steg 1, pluss query-parameter (`status_query_token`) du fikk i steg 2. Hvis signeringsoppdraget er lagt på en spesifikk kø, så må også headeren `polling-queue` settes til navnet på køen.
```

Comment:
Jeg har kommentert at headeren bør hete `Polling-Queue` for å følge navnekonvensjoner, husk i så fall å oppdatere her også 💡

Reply:
Kommentert her altså: https://github.com/digipost/signature-api-client-java/pull/59#discussion_r157733027